persistence/sql/migrations/sql/20241001000000000001_identity_recovery_addresses_pk.mysql.down.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE identity_recovery_codes
+    DROP FOREIGN KEY identity_recovery_codes_identity_recovery_addresses_id_fk,
+    DROP FOREIGN KEY identity_recovery_codes_identity_id_fk;
+
+ALTER TABLE identity_recovery_tokens
+    DROP FOREIGN KEY identity_recovery_tokens_identity_id_fk,
+    DROP FOREIGN KEY identity_recovery_tokens_identity_recovery_addresses_id_fk;
```

Comment:
Do we need to remove/recreate the child's foreign key if we change a constraint in the other table?

Reply:
This dance is only necessary on MySQL.